python/src/trezorlib/cardano.py

```diff
@@ -36,8 +36,8 @@
     "owners",
 )
 REQUIRED_FIELDS_WITHDRAWAL = ("path", "amount")
-REQUIRED_FIELDS_TOKEN_BUNDLE = ("policy_id", "token_amounts")
-REQUIRED_FIELDS_TOKEN_AMOUNT = ("raw_asset_name", "amount")
+REQUIRED_FIELDS_TOKEN_GROUP = ("policy_id", "tokens")
+REQUIRED_FIELDS_token = ("asset_name_bytes", "amount")
```

Comment:
`REQUIRED_FIELDS_token` - probably a result of a replace. The `_token` isn't capitalized.

Reply:
:facepalm: